docs/3.0/resources/whats-new-prefect-3.mdx

```diff
@@ -1,15 +1,21 @@
 ---
-title: What's new in Prefect 3
+title: What's new in Prefect 3.0
 sidebarTitle: What's new
 ---
 
-Prefect 3 represents a significant leap forward in workflow orchestration, bringing a host of new features, performance improvements, and expanded capabilities to enhance your data engineering experience. Let's explore the exciting new additions and enhancements in this release.
+Prefect 3.0 is now the default open source version!
+
+Prefect 3.0 represents a significant leap forward in workflow orchestration, bringing a host of new features, performance improvements, and expanded capabilities to enhance your data engineering experience. Let's explore the exciting new additions and enhancements in this open source release.
```

Comment:
same comment on both of these sentences as above.

Reply:
edited locally and pushing